torch/fx/symbolic_trace.py

```diff
@@ -12,6 +15,17 @@
 
 HAS_VARSTUFF = inspect.CO_VARARGS | inspect.CO_VARKEYWORDS
 
+# These need to run in global scope to handle nested calls correctly
+_orig_module_call : Callable = torch.nn.Module.__call__
+_orig_module_getattr : Callable = torch.nn.Module.__getattr__
+
+# Functions we will eagerly wrap when we see them while tracing
+# this captures both `math.sqrt()` and `from math import sqrt` automatically
+_autowrap_search : List[dict] = [math.__dict__]
+_autowrap_function_ids : Set[int] = {id(value) for name, value in math.__dict__.items()
+                                     if not name.startswith("_") and callable(value)}
```

Comment:
One thing I tried to convey in our meeting but don't know if I got through was: can we make this configurable from the `symbolic_trace`/`Tracert.trace` APIs. E.g. take a parameter of `List[module]` or `List[str]` that refers to modules that the user wants to be patched automatically. This could be `[math]` by default. This way, this behavior is obvious from the API and is also configurable

Reply:
You prefer an option to Tracer() as opposed to extending the wrap() API?

Reply:
I moved it to an option to Tracer()